docs/CHANGELOG.md

```diff
@@ -1,38 +1,61 @@
+## Version ["v0.7.0"] - 2024-04-04
+
+### Features

```

Comment:
The changelogs are actually generated by git-cliff. This looks much nicer though. I am not sure if it's worth setting up git-cliff properly again.